### ISSUE TITLE
PS 1.7.6.7 missing option to switch from net prices to gross prices

### ISSUE THREAD

**Issue by icemansparks** [PrestaShop/ps_googleanalytics#57]
Currently the submitted value to google analytics always contains the net price value for a product. 
Is there any possibility for a switch so that always the product gross price is submitted instead?

**Comment by matks** [comment]
That would be advanced module configuration.

So if I follow your mind, it would be some "advanced configuration" settings, where you can see the list of data being sent to GA (product data, order data, user data ...) and you can configure some of them.

For example
- for product you can choose "send net value" or "send gross price value"
- for order you can choose "send tax included total" or "send tax excluded total"

I'm wondering though ... maybe there is a way to send _all data_ to GA and then, inside GA, you can configure what you monitor ? 🤔

**Comment by icemansparks** [comment]
Yes this is what i had in mind. But since i am not a Google Analytics professional, there may be better solutions to that.
I just set up the module like you are supposed to, activated GA and could see the difference in the prices. Thats how i figured that only net prices are submitted.

**Comment by Hlavtox** [comment]
@icemansparks GA implementation is done exactly as Google's documentation says, and the whole ecosystem counts on the data being in some format... You could have some wrong metrics here and there and stuff like this.

Don't take it wrong, but I think if you want to change it, you should do it only for yourself. :-)
